src/Compilers/CSharp/Portable/CodeGen/EmitArrayInitializer.cs

```diff
@@ -372,9 +372,23 @@ private static bool IsMultidimensionalInitializer(ImmutableArray<BoundExpression
 
 #nullable enable
 
+        /// <summary>Tries to emit a ReadOnlySpan construction as a wrapper for a blob rather than as a wrapper for an array construction.</summary>
+        /// <param name="spanType">The type of the span being constructed.</param>
+        /// <param name="wrappedExpression">The expression being wrapped in a span.</param>
+        /// <param name="used">true if the result of the expression is used; false if it's required only for its side effects.</param>
+        /// <param name="inPlaceTarget">A non-null expression if the construction is initializing an existing local in-place; otherwise, null.</param>
+        /// <param name="avoidInPlace">An output Boolean indicating whether a caller trying to perform in-place initialization should instead prefer to assign the local to a new value.</param>
```

Comment:
What does in-place initialization mean exactly?

Reply:
Assigning to an existing local.